include/swift/Remote/MemoryReader.h

```diff
@@ -148,19 +148,32 @@ class MemoryReader {
     return RemoteAbsolutePointer("", readValue);
   }
 
-  /// Atempt to resolve the pointer to a symbol for the given remote address.
   virtual llvm::Optional<RemoteAbsolutePointer>
   resolvePointerAsSymbol(RemoteAddress address) {
     return llvm::None;
   }
```

Comment:
I have kept this in to allow this change to allow this PR to be free of a dependent change in llvm-project. Later, it will be removed.